src/Page.test.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+
+import { render, fireEvent, screen } from '@testing-library/react';
+
+import Page from './Page';
+
+const propsForTest = {
+  taskTitle: 'hello world',
+  onChangeTitle: jest.fn(),
+  onClickAddTask: jest.fn(),
+  tasks: [
+    {
+      id: 1,
+      title: '뭐라도 하기',
+    },
+    {
+      id: 2,
+      title: '뭐라도 안하기',
+    },
+  ],
+  onClickDeleteTask: jest.fn(),
+};
+
+describe('Page 컴포넌트의', () => {
+  const {
+    taskTitle, tasks, onChangeTitle, onClickAddTask, onClickDeleteTask,
+  } = propsForTest;
+
+  beforeEach(() => {
+    render(<Page
+      taskTitle={taskTitle}
+      tasks={tasks}
+      onChangeTitle={onChangeTitle}
+      onClickAddTask={onClickAddTask}
+      onClickDeleteTask={onClickDeleteTask}
+    />);
+  });
```

Comment:
```suggestion
describe('...', () => {

  const tasks = [...];
  
  function renderPage() {
    return render((
      <Page
        tasks={tasks}
        taskTitle=""
        onChangeTitle={handleChangeTitle}
        onClickAddTask={handleClickAddTask}
        onClickDelete={handleClickDeleteTask}
      />
    ));
  }
```

각 테스트에서 무슨 일이 일어나고 있는지 파악하기 힘드네요.

우선 BDD (describe-context-it) 에 맞게 Page 테스트를 개선해보시고

위에 제가 제안드린 코드는 어떤지 한번 생각해보시면 좋을 것 같아요.